src/source/CurlApiCallbacks.c

```diff
@@ -42,6 +42,9 @@ STATUS createCurlApiCallbacks(PCallbacksProvider pCallbacksProvider, PCHAR regio
     // Endpoint period will be the same for all streams
     pCurlApiCallbacks->cacheUpdatePeriod = endpointCachingPeriod;
 
+    // API call caching type
+    pCurlApiCallbacks->cacheType = cacheType;

```

Comment:
Storing the caching type so we can use it later as we end up with a single callback entries few lines below